static/js/contact.js

```diff
@@ -8,8 +8,10 @@ const errorEl = 'contact-error';
 form?.addEventListener('submit', async (event) => {
   event.preventDefault();
   
+  // TODO: Restore the reCAPTCHA implementation
```

Comment:
This has been removed after encountering [this](https://cloud.google.com/recaptcha/docs/authentication) issue. It will be handled in a future sprint.